web/src/admin/groups/RelatedUserList.ts

```diff
@@ -175,10 +202,10 @@ export class RelatedUserList extends WithBrandConfig(WithCapabilitiesConfig(Tabl
         const disabled = this.selectedElements.length < 1;
         return html`<ak-forms-delete-bulk
             objectLabel=${msg("User(s)")}
-            actionLabel=${msg("Remove Users(s)")}
+            actionLabel=${msg("Remove User(s)")}
             action=${msg("removed")}
             actionSubtext=${msg(
-                str`Are you sure you want to remove the selected users from the group ${this.targetGroup?.name}?`,
+                str`Are you sure you want to remove the selected users from ${this.targetGroup?.name || this.targetRole?.name}?`,
             )}
```

Comment:
Add fallback for no target e.g.
```suggestion
  const targetLabel = this.targetGroup?.name || this.targetRole?.name
  ...
           actionSubtext=${targetLabel ? msg(
                str`Are you sure you want to remove the selected users from ${targetLabel}?`) : msg("Are you sure you want to remove the selected users?"),
}
```